### PR TITLE
Display media content status

### PR DESCRIPTION
**Pull request by devkonrad**
Display media content status

Changes:
modified: app/controllers/admin/paper_media_consents_controller.rb
modified: app/views/admin/participants/show.html.erb
modified: config/routes.rb 

Refs: https://github.com/Iridescent-CM/technovation-app/issues/3272

### CODE REVIEW

File: app/views/admin/participants/show.html.erb [hunk 1]
```diff
@@ -200,6 +200,18 @@
       </div>
     <% end %>
 
+    <% if @account.student_profile %>
+      <div class="panel">
+        <h5>
+            <% if @account.student_profile&.media_consent&.consent_provided %>
```
Comment (shaunxp20) [inline app/views/admin/participants/show.html.erb:206]: `@account.student_profile&.` - it doesn't look like the `&` is necessary here, on line 203, we're already checking for a `student_account`

And another minor thing, it looks like the indentation is off for this entire `if` block. 🧐 😄

File: app/views/admin/participants/show.html.erb [hunk 2]
```diff
@@ -200,6 +200,18 @@
...
+              Media Consent on file
```
Comment (shaunxp20) [inline app/views/admin/participants/show.html.erb:207]: It seems like this language might not quite be accurate - the conditional on line 206 checks for consent, not if it's "on file" or not. 🤔 😄

Comment (devkonrad) [reply app/views/admin/participants/show.html.erb:207]: I agree. @rsgonzal , is the purpose of this feature is check if the consent is provided?

**Comment by dboyer** [comment]
Update after discussion with the program team. Ideally, they would like to know both if media consent has been given and the format it was given in (i.e. via the online form or via a paper form that someone emailed). So there are five possible states:

Agree to media consent via online form
Agree to media consent via paper form
Do not agree to media consent via online form
Do not agree to media consent via paper form
Has not completed media consent form

@devkonrad I realize this has expanded from the original plan for the issue. Please feel free to check in with someone if you need more details about how this could be implemented.

**Comment by devkonrad** [comment]
As i could see, currently we have two params that controls the media_consent status inside de platform (consent_provided boolean flag and electronic_signature text info), so just por confirmation about the 5 status thar @dboyer mentioned:

Agree to media consent via online form => consent_provided is TRUE and electronic_signature is "Guardian Name"    
Agree to media consent via paper form => consent_provided is TRUE and electronic_signature is "ON FILE"
Do not agree to media consent via online form  => consent_provided is FALSE and electronic_signature is "Guardian Name"
Do not agree to media consent via paper form => consent_provided is FALSE and electronic_signature is "On File"
Has not completed media consent form  => The is no Media Content for the Student

Is that correct?

**Comment by viviancan** [comment]
hey alex! Just did a quick QA locally! I think some of the wording/states are missing. Roxana will provide updated screens/details a little later. Thanks!

**Review by viviancan — CHANGES_REQUESTED** [review]
hey alex! Just did a quick QA locally! I think some of the wording/states are missing. Roxana will provide updated screens/details a little later. Thanks!

**Comment by devkonrad** [comment]
> hey alex! Just did a quick QA locally! I think some of the wording/states are missing. Roxana will provide updated screens/details a little later. Thanks!

Hi, Vivian. Thanks for the feedback. I'll wait for @rsgonzal screen.

**Pull request closed without merging**